NEWS.md

```diff
@@ -1,5 +1,9 @@
 # EpiNow2 1.4.9000
 
+## Package
+
+* Reduced the number of long-running examples.
```

Comment:
Missing the standard PR, author and reviewer attribution we have used elsewhere.